conf/plugin-templates/common/s3-storage-plugin.template

```diff
@@ -0,0 +1,124 @@
+{
+  "name" : "s3",
+  "config" : {
+    "type": "file",
+    "enabled": true,
+    "connection": "s3a://tstbucket",
+    "config": {
+      "fs.s3a.endpoint": "http://localhost:9000",
+      "fs.s3a.connection.ssl.enabled": "false",
+      "fs.s3a.path.style.access": "true",
+      "fs.s3a.access.key": "ZYVR33VM12IAHETIJYF9",
+      "fs.s3a.secret.key": "BjGb9WpNd74t5T7FcBlXRgUcSyq505G598T8Yxg6",
```

Comment:
Are these okay to be public?

Reply:
It isn't a real Amazon S3 credentials. I just choose this random keys and hardcode it for the Minio server.